### ISSUE TITLE
Expose option to append SMARTS and atom type to default list

### ISSUE THREAD

**Issue by diogomart** [forlilab/Meeko#54]
Something like the following, which is easier than getting the defaults and passing the full set of parameters to `atom_type_smarts`.

```python
mk_prep = MoleculePreparation(append_types='[{"SMARTS": "[Oh1][c]", "atype": "OX"}]')
```

Using JSON makes it easy to do the same from `mk_prepare_ligand.py`

**Comment by althea-hansel** [comment]
Created pull request

**Comment by diogomart** [comment]
implemented in #56
